test/chainnotifier_mock.go

```diff
@@ -0,0 +1,160 @@
+package test
```

Comment:
Perhaps this code should've also been moved to `lndclient` so that we don't have to copy it everywhere?

Reply:
There was quite some discussion around this in loop. The server also used that test code and imported it from the client repo. But then the server required the mocks to be more specific and have more features, so every time someone wanted to extend the tests in the server repo a change in the client repo was also necessary. That's why it was instead copied to the server and no longer imported.

Moving the code to `lndclient` doesn't really solve the problem of having different requirements for the mocks in different projects.